src/libmongoc/src/mongoc/mongoc-client.c

```diff
@@ -3080,3 +3081,40 @@ mongoc_client_enable_auto_encryption (mongoc_client_t *client,
    }
    return _mongoc_cse_client_enable_auto_encryption (client, opts, error);
 }
+
+bool
+mongoc_client_set_timeout_ms (mongoc_client_t *client,
+			      int64_t timeout,
+			      bson_error_t *error) {
+   if (!client) {
+      bson_set_error (error,
+		      MONGOC_ERROR_CLIENT,
+		      MONGOC_ERROR_CLIENT_NULL,
+		      "NULL client parameter");
+	 return false;		      
+   }
```

Comment:
For consistency, we should `BSON_ASSERT` the client exists like the other functions in this file.

Reply:
Changed.